gradle-nonnull-plugin/plugin/src/test/groovy/com/novoda/gradle/nonnull/AndroidProjectIntegrationTest.groovy

```diff
@@ -68,6 +67,24 @@ public class AndroidProjectIntegrationTest {
         assertThat(task.outcome).isEqualTo(TaskOutcome.SUCCESS)
     }
 
+    @Test
+    public void shouldNotGeneratePackInfoWhenItsAlreadyPartOfSources() {
```

Comment:
`shouldNotGeneratePackInfoWhenItsAlreadyPartOfSources` should be `shouldNotGeneratePackageInfoWhenItsAlreadyPartOfSources` to avoid the `Pack` abbreviation.